CHANGELOG.md

```diff
@@ -159,12 +159,20 @@ For more information about this file see also [Keep a Changelog](http://keepacha
 - Added PEcAn.utils::download.file() to allow for use of alternative FTP programs
 - Updated downloadAmeriflux and downloadNARR to make use of PEcAn.utils::download.file()
 - Added -w flag to load.bety.sh script to specify the URL to fetch the data from
+<<<<<<< HEAD
```

Comment:
need to resolve merge conflict and also make sure your changes are in the CHANGELOG

Reply:
Ok. Noted. Will do for next time.